src/t8_vtk/t8_vtk_reader.cxx

```diff
@@ -25,7 +25,8 @@ along with t8code; if not, write to the Free Software Foundation, Inc.,
 #include "t8_vtk_polydata.hxx"
 #include "t8_vtk_parallel.hxx"
 #include "t8_vtk_types.h"
-#include <t8_geometry/t8_geometry_implementations/t8_geometry_linear.h>
+#include <t8_cmesh.hxx>
+#include <t8_geometry/t8_geometry_implementations/t8_geometry_linear.hxx>
```

Comment:
Still don't know the difference, but keeping it constant troughout a file seems to be useful for the compiler